tests/test_ptc.py

```diff
@@ -260,193 +255,25 @@ def test_covAstier(self):
-                    17.40755051,
-                ],
-            ]
+            [[8.99474598, 9.94916264, -27.90587299],
+             [-2.95079527, -17.11827641, -47.88156244],
+             [5.24915021, -3.25786165, 26.09634067]],
         )
-
         noiseMatrixNoBExpected = np.array(
-            [
-                [
-                    6.53126505,
-                    12.14827594,
-                    -37.11919923,
-                    41.18675353,
-                    -85.1613845,
-                    -28.45801954,
-                    -61.24442999,
-                    88.76480122,
-                ],
-                [
-                    -4.64541165,
-                    -23.20924747,
-                    -66.08733987,
-                    -0.87558055,
-                    12.20111853,
-                    24.84795549,
-                    -34.92458788,
-                    24.42745014,
-                ],
-                [
-                    7.66734507,
-                    -4.51403645,
-                    35.69834113,
-                    52.73693356,
-                    -30.85044089,
-                    10.86761771,
-                    10.8503068,
-                    -2.18908327,
-                ],
-                [
-                    50.9901156,
-                    -7.34803977,
-                    5.33443765,
-                    21.60899396,
-                    -25.06129827,
-                    15.14015505,
-                    10.94263771,
-                    29.23975515,
-                ],
-                [
-                    -48.66912069,
-                    -31.58003774,
-                    21.81305735,
-                    -13.08993444,
-                    8.17275394,
-                    74.85293723,
-                    -11.18403252,
-                    -31.7799437,
-                ],
-                [
-                    -38.55206382,
-                    22.92982676,
-                    13.39861008,
-                    -33.3307362,
-                    8.65362238,
-                    29.18775548,
-                    31.78433947,
-                    1.27923706,
-                ],
-                [
-                    23.33663918,
-                    -41.74105625,
-                    -26.55920751,
-                    -24.71611677,
-                    12.13343146,
-                    11.25763907,
-                    21.79131019,
-                    -26.579393,
-                ],
-                [
-                    11.44334226,
-                    -34.9759641,
-                    13.96449509,
-                    19.64121933,
-                    -36.09794843,
-                    -34.27205933,
-                    -25.16574105,
-                    23.80460972,
-                ],
-            ]
+            [[8.71049338, 12.48584043, -37.06585088],
+             [-4.80523971, -23.29102809, -66.37815343],
+             [7.48654766, -4.10168337, 35.64469824]],
```

Comment:
In the checks below I think we should have explicit tests of the `noise` and `noiseErr`.  The `noise` is implicitly handled by the no-warnings, but it's good to make it explicit.  Plus we need the error test for the full fit, not just the polynomial and expapproximation done later.